rbac/enforcer.go

```diff
@@ -165,7 +165,17 @@ func PermsForUser(user string) ([]policy.Permission, error) {
 	}
 	var s []policy.Permission
 	for _, perm := range append(perms, implicit...) {
-		s = append(s, policy.NewPermission(perm))
+		p := policy.NewPermission(perm)
```

Comment:
What does the UI look like for this ?

Reply:
Before
<img width="1422" height="768" alt="image" src="https://github.com/user-attachments/assets/c3881012-a90c-4a38-8115-1ba88f7c6919" />


After
<img width="1422" height="768" alt="image" src="https://github.com/user-attachments/assets/e8ab35a2-e770-425d-a68f-636605040ec2" />